### DISCUSSION TITLE
Question: Is there any way to change icon of navbaritem when it is selected in sidebar?

### DISCUSSION THREAD

**Discussion by praveen7557** [InboxSDK/InboxSDK#925] [General]
If you see the inbox icon in the sidebar, we can see the icon update based on whether it's selected or not.

<img width="248" alt="Screenshot 2023-01-05 at 7 53 35 AM" src="https://user-images.githubusercontent.com/13882201/210687305-58735536-3690-4ec5-8c62-f3ba654b39e2.png">

<img width="248" alt="Screenshot 2023-01-05 at 7 53 30 AM" src="https://user-images.githubusercontent.com/13882201/210687336-a8bf863e-3d69-4ce5-9b8a-f77b7df8e26e.png">

can we achieve the same with inboxSdk? currently, I'm using the below method to add the nav item

```
rootNavItem.addNavItem({
   name: 'All Tracked Emails',
   routeID: 'tracked',
   iconUrl:
     chrome.runtime.getURL('img/gmail-thread-view/tracked-mails.svg'),
})
```

**Comment by arthur-er** [comment]
You need to use one of the observer methods to update (kefir, rxjs)
